[error] lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-featurespec_2.13/3.2.5/scalatest-featurespec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-featurespec_2.13/3.2.5/scalatest-featurespec_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-compatible/3.2.5/scalatest-compatible-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-compatible/3.2.5/scalatest-compatible-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-core_2.13/3.2.5/scalatest-core_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-core_2.13/3.2.5/scalatest-core_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-funspec_2.13/3.2.5/scalatest-funspec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-funspec_2.13/3.2.5/scalatest-funspec_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-propspec_2.13/3.2.5/scalatest-propspec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-propspec_2.13/3.2.5/scalatest-propspec_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-wordspec_2.13/3.2.5/scalatest-wordspec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-wordspec_2.13/3.2.5/scalatest-wordspec_2.13-3.2.5.jar
[error] 
[error] 	at lmcoursier.internal.shaded.coursier.Artifacts$.$anonfun$fetchArtifacts$14(Artifacts.scala:302)
[error] 	at lmcoursier.internal.shaded.coursier.util.Task$.$anonfun$flatMap$2(Task.scala:14)
[error] 	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
[error] 	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
[error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[error] 	at java.base/java.lang.Thread.run(Thread.java:832)
[error] Caused by: lmcoursier.internal.shaded.coursier.cache.ArtifactError$DownloadError: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-featurespec_2.13/3.2.5/scalatest-featurespec_2.13-3.2.5.jar
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:1018)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1032)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:320)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:507)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:59)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:84)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:508)
[error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
[error] 	at scala.util.Success.map(Try.scala:213)
[error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[error] 	at java.base/java.lang.Thread.run(Thread.java:832)
[error] Caused by: java.net.UnknownHostException: repo1.maven.org
[error] 	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:567)
[error] 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:333)
[error] 	at java.base/java.net.Socket.connect(Socket.java:648)
[error] 	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:290)
[error] 	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:173)
[error] 	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:182)
[error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)
[error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)
[error] 	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:265)
[error] 	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:372)
[error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:177)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1194)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1072)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1070)
[error] 	at java.base/java.security.AccessController.doPrivileged(AccessController.java:691)
[error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:934)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1069)
[error] 	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:163)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1600)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1520)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1518)
[error] 	at java.base/java.security.AccessController.doPrivileged(AccessController.java:691)
[error] 	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:934)
[error] 	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1517)
[error] 	at java.base/java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:527)
[error] 	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:308)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirectTo(CacheUrl.scala:152)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.redirect(CacheUrl.scala:162)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:304)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheUrl$.urlConnectionMaybePartial(CacheUrl.scala:277)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$33(FileCache.scala:349)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.$anonfun$downloading$1(FileCache.scala:998)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withUrlLock(CacheLocks.scala:102)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.helper$2(FileCache.scala:998)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache$.coursier$cache$FileCache$$downloading(FileCache.scala:1032)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.doDownload$1(FileCache.scala:320)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$54(FileCache.scala:507)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:59)
[error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:84)
[error] 	at lmcoursier.internal.shaded.coursier.cache.FileCache.$anonfun$download$32(FileCache.scala:508)
[error] 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[error] 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
[error] 	at scala.util.Success.map(Try.scala:213)
[error] 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[error] 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[error] 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[error] 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[error] 	at java.base/java.lang.Thread.run(Thread.java:832)
[error] (app / ssExtractDependencies) lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-featurespec_2.13/3.2.5/scalatest-featurespec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-featurespec_2.13/3.2.5/scalatest-featurespec_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-compatible/3.2.5/scalatest-compatible-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-compatible/3.2.5/scalatest-compatible-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-core_2.13/3.2.5/scalatest-core_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-core_2.13/3.2.5/scalatest-core_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-funspec_2.13/3.2.5/scalatest-funspec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-funspec_2.13/3.2.5/scalatest-funspec_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-propspec_2.13/3.2.5/scalatest-propspec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-propspec_2.13/3.2.5/scalatest-propspec_2.13-3.2.5.jar
[error] https://repo1.maven.org/maven2/org/scalatest/scalatest-wordspec_2.13/3.2.5/scalatest-wordspec_2.13-3.2.5.jar: download error: Caught java.net.UnknownHostException: repo1.maven.org (repo1.maven.org) while downloading https://repo1.maven.org/maven2/org/scalatest/scalatest-wordspec_2.13/3.2.5/scalatest-wordspec_2.13-3.2.5.jar
